ignite/contrib/metrics/regression/median_absolute_error.py

```diff
@@ -31,6 +31,15 @@ class MedianAbsoluteError(_BaseRegressionEpoch):
 
     __ https://arxiv.org/abs/1809.03006
 
```

Comment:
It misses `device` argument in `__init__`

Reply:
Removed `device`